runtime/bcutil/cfreader.c

```diff
@@ -1947,11 +1947,17 @@ checkAttributes(J9CfrClassFile* classfile, J9CfrAttribute** attributes, U_32 att
 					goto _errorFound;
 				}
 
+				if ((flags & CFR_Xfuture) && (bcvCheckName(&cpBase[value]))) {
```

Comment:
Can you refactor this a bit so there's a single common check for `(cpBase)` and then the UTF8 check and the name check under it?

```c
if (cpBase) {
  if (cpBase[value].tag != CFR_CONSTANT_Utf8) {
      errorCode = J9NLS_CFR_ERR_LOCAL_VARIABLE_NAME_NOT_UTF8__ID;
      goto _errorFound;
   }
   if (bcvCheckName(&cpBase[value])) {
      ......
      goto _errorFound;
   }
}

```

Reply:
updated the pattern in a few places.